PokeAlarm/Filters/RaidFilter.py

```diff
@@ -126,6 +138,14 @@ def to_dict(self):
         if self.gym_name_contains is not None:
             settings['gym_name_matches'] = self.gym_name_contains
 
+        # Gym Sponsor
+        if self.is_sponsor is not None:
+            settings['gym_is_sponsor'] = self.is_sponsor
```

Comment:
Update to `is_sponsor` to match the attribute name.

Reply:
I made it different on purpose as all the other values here are different, despite having no idea why?!